daft-parquet/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+TRIES=3
+QUERY_COUNT=43
+RESULT_FILE="results/c6a.metal.json"
```

Comment:
L. 5 says `c6a.metal`, l. 31 says `c6a.4xlarge`.

Reply:
Update run.sh to accept 'c6a.metal' and 'c6a.4xlarge' as parameters